src/main.py

```diff
@@ -170,6 +173,17 @@ def get_cluster_run_args():
         default=2,
         type=int
     )
+    parser.add_argument(
+        '--failure_rate',
+        default=0.2,
+        type=float
+    )
+    parser.add_argument(
+        '--recover_rate',
+        default=0.2,
+        type=float
+    )
```

Comment:
does 0.2 recovery rate mean 20% of the nodes will crash every __ seconds/minutes/hour? Please add some documentation in a way it is understandable.